### PR TITLE
Bump version: 0.9.2 → 0.10.0

### PR DESCRIPTION
**Pull request by simonafk**

### CODE REVIEW

File: docs/release-notes.rst [hunk 1]
```diff
@@ -74,11 +74,8 @@ Upcoming Version (Not Yet Released)
    in the "stable" docs (corresponding to the last release) but will be visible in the
    "latest" docs (corresponding to the master branch).
 
-Bug Fixes
-.........
-
-- Fixed a bug where Bionic could crash when using parallel execution after manually
-  deleting some persisted artifacts (such as with the :ref:`Cache API <cache-api>`).
+0.10.0 (Feb 12, 2021)
```
Comment (namanjain) [inline docs/release-notes.rst:77]: Let's comment out the `Upcoming Version` heading above so it doesn't appear in stable docs.

**Review by namanjain — APPROVED** [review]
Ship it!

**Comment by jqmp** [comment]
It looks like the docs are currently not building. You can tell by checking out the [latest](https://bionic.readthedocs.io/en/latest/index.html) version of the docs, which doesn't have the most recent docs change for AIP. This is a good thing to double-check before releasing a version.

I'm not sure what the build failure is. I'm going to try to re-run the build and also check whether @simonafk or @namanjain have access to the Read The Docs account.

**Comment by jqmp** [comment]
It looks like @namanjain already has access to Bionic's configuration at [Read The Docs](https://readthedocs.org/projects/bionic/). @simonafk, if you have a Read The Docs account I can add you too.

**Comment by simonafk** [comment]
> It looks like @namanjain already has access to Bionic's configuration at [Read The Docs](https://readthedocs.org/projects/bionic/). @simonafk, if you have a Read The Docs account I can add you too.

@namanjain I am using github to login into Read The Docs.

**Comment by jqmp** [comment]
> > It looks like @namanjain already has access to Bionic's configuration at [Read The Docs](https://readthedocs.org/projects/bionic/). @simonafk, if you have a Read The Docs account I can add you too.
> 
> Yes, my account is the same as that of Github.

Great, I just added you.

**Pull request merged**